enos/enos-scenario-upgrade.hcl

```diff
@@ -91,6 +92,14 @@ scenario "upgrade" {
       cert_file   = step.provision_cluster.cert_file
       key_file    = step.provision_cluster.key_file
       nomad_token = step.provision_cluster.nomad_token
+      workloads = {
+        service_raw_exec = { job_spec = "jobs/raw-exec-service.nomad.hcl", alloc_count = 3, type = "service" }
+        service_docker   = { job_spec = "jobs/docker-service.nomad.hcl", alloc_count = 3, type = "service" }
+        system_docker    = { job_spec = "jobs/docker-system.nomad.hcl", alloc_count = 0, type = "system" }
```

Comment:
The alloc count for system jobs doesn't really matter, its set to 0 to emphasis it